Yank/restraints.py

```diff
@@ -2117,18 +2123,23 @@ class RMSD(ReceptorLigandRestraint):
     ...                  K_RMSD=1.0*unit.kilocalories_per_mole/unit.angstrom**2,
     ...                  RMSD0=1*unit.angstroms)
 
+    Find missing parameters
+
+    >>> restraint.determine_missing_parameters(thermodynamic_state, sampler_state, topography)
+
     Get standard state correction.
 
     >>> correction = restraint.get_standard_state_correction(thermodynamic_state)
 
     """
     def __init__(self, restrained_receptor_atoms=None, restrained_ligand_atoms=None,
-                 K_RMSD=None, RMSD0=None):
+                 K_RMSD=None, RMSD0=None,
+                 reference_state=None):
         self.restrained_receptor_atoms = restrained_receptor_atoms
         self.restrained_ligand_atoms = restrained_ligand_atoms
         self.K_RMSD = K_RMSD
         self.RMSD0 = RMSD0
-        self._reference_sampler_state = None
+        self.reference_state = reference_state
```

Comment:
Thanks!

I think `reference_state` is a little ambiguous though. I've had somebody trying to use the API complaining in an email that handling all these "states" can be confusing at times. I would either call this `reference_sampler_state` or just pass the positions and call it `reference_positions`.

Reply:
I thought about allowing either a `SamplerState` or numpy array, but then I figured it would be better to minimize the number of objects users are allowed to and should pass around. Enforcing sampler state ensures a certain structure.

I'll change the name